x/pairing/keeper/scores/geo_req.go

```diff
@@ -38,13 +42,16 @@ func (gr GeoReq) Score(score PairingScore) math.Uint {
 	return cost
 }
 
-func (gr GeoReq) GetName() string {
+func (gr *GeoReq) GetName() string {
```

Comment:
why do we make it pointers?

Reply:
reverted. see https://github.com/lavanet/lava/commit/b2c155b44d7902f6ba4371cdbcb58ad1d51caea6